src/SqlPersistence.Tests/APIApprovals.Approve.approved.txt

```diff
@@ -215,6 +215,12 @@ namespace NServiceBus
         public static NServiceBus.Persistence.Sql.SubscriptionSettings SubscriptionSettings(this NServiceBus.PersistenceExtensions<NServiceBus.Persistence.Sql.SqlPersistence> configuration) { }
         public static void TablePrefix(this NServiceBus.PersistenceExtensions<NServiceBus.Persistence.Sql.SqlPersistence> configuration, string tablePrefix) { }
     }
+    public class static SqlPersistenceOutboxSettingsExtensions
+    {
+        public static void DisableCleanup(this OutboxSettings configuration) { }
+        public static void SetFrequencyToRunDeduplicationDataCleanup(this OutboxSettings configuration, TimeSpan frequencyToRunDeduplicationDataCleanup) { }
```

Comment:
Can we tweak the settings names a bit? How about:
* `DeduplicationDataCleanupInterval`
* `DeduplicationDataRetentionPeriod`

or maybe if we want those to be imperative as `DiableCleanup` we could use:
* `RunDeduplicationDataCeanupEvery`
* `KeepDeduplicationDataFor`

Reply:
We can, I tried to be more consistent with names in other persistence, but I agree they could be better :/ So should we prioritize consistency or readability?

Reply:
I am :+1: for the imperative versions.

Reply:
OK, consistency is anyway not possible to achieve right now